app/components/Post/UserInfo.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import { View, Text, StyleSheet } from 'react-native'
+import Moment from 'moment'
+
+import { styles } from '../../res/css'
+import ShockAvatar from '../ShockAvatar'
+
+const style = StyleSheet.create({
+  userInfoContainer: {
+    flexDirection: 'row',
+    alignItems: 'center',
+    width: '100%',
+    padding: 10,
+  },
+  userInfoTextContainer: {
+    width: '100%',
+    justifyContent: 'space-between',
+    padding: 5,
+  },
+  authorDisplayNameStyle: {
+    ...styles.textBold,
```

Comment:
Not a mixin but a whole stylesheet in and of itself, need to be grouped with an stylsheet of your own through either array concatenation or StyleSheet.flatten